test/column/draggable.test.ts

```diff
@@ -1,31 +1,36 @@
-import { fireEvent } from '@testing-library/react';
+import { act } from 'react';
+import { userEvent } from '@vitest/browser/context';
 
 import type { Column } from '../../src';
 import { getHeaderCells, setup } from '../utils';
 
 const columns: readonly Column<never>[] = [
   {
     key: 'col1',
-    name: 'col1'
+    name: 'col1',
+    width: 100
```

Comment:
Tests fail with `width`

Reply:
Tests pass without `width` if you add `viewport: { width: 1920, height: 1080 }`

![image](https://github.com/user-attachments/assets/00dfbe1c-3246-43f9-afb1-82886f81444e)

It might be good to add it so we test something that better represent the average viewport.